eth/db/header.py

```diff
@@ -46,6 +55,40 @@ class HeaderDB(HeaderDatabaseAPI):
     def __init__(self, db: AtomicDatabaseAPI) -> None:
         self.db = db
 
+    def get_header_chain_gaps(self) -> ChainGaps:
+        return self._get_header_chain_gaps(self.db)
+
+    @classmethod
+    def _get_header_chain_gaps(cls, db: DatabaseAPI) -> ChainGaps:
+        try:
+            encoded_gaps = db[SchemaV1.make_header_chain_gaps_lookup_key()]
+        except KeyError:
+            return GENESIS_CHAIN_GAPS
```

Comment:
@carver I want to point out this change. Previously we were just returning an empty tuple here but with the updated datastructure and empty tuple doesn't match the type anymore which is why I came up with `GENESIS_CHAIN_GAPS`. On the plus side, this slightly improves the `calculate_gaps` logic.